lib/BackbeatConsumer.js

```diff
@@ -511,10 +514,38 @@ class BackbeatConsumer extends EventEmitter {
     }
 
     /**
-    * force commit the current offset and close the client connection
-    * @param {callback} cb - callback to invoke
-    * @return {undefined}
-    */
+     * pause the kafka consumer
+     * @param {string} site - name of site
+     * @return {undefined}
+     */
+    pauseService(site) {
+        // Use of KafkaConsumer#pause did not work. Using alternative
+        // of unsubscribe/subscribe
+        if (this._consumerReady) {
+            this._consumer.unsubscribe();
+            this._consumerReady = false;
+            this._log.info(`paused replication for site ${site}`);
```

Comment:
Similarly, we're not bound to replication service here. We may log something here (preferably debug) but this log would go into the service code (QueueProcessor) instead.